conf/config-default.yaml

```diff
@@ -600,7 +600,9 @@ plugin_attr:          # Plugin attributes
     #   - 100
     #   - 200
     #   - 500
-    # expire: 86400                         # the expiration time after metrics become inactive, unit: second
+    # expire: 0                       # The expiration time after metrics become inactive, unit: second.
+                                      # 0 means the metrics will not expire
+                                      # We recommend using 600 If you need to set the expiration time
```

Comment:
should avoid to use `we`

How about this:

If you need to set the expiration time, it is recommended to use 600, which is 10 minutes.

Reply:
Done